libraries/node-core-library/src/IPackageJson.ts

```diff
@@ -68,6 +68,7 @@ export interface IPeerDependenciesMetaTable {
 export interface IDependenciesMetaTable {
   [dependencyName: string]: {
     injected?: boolean;
+    [key: string]: unknown;
```

Comment:
Instead of duplicating this in libraries/rush-lib/src/api/PackageJsonEditor.ts, can you just extract this object to its own interface and use that in rush-lib?